### DISCUSSION TITLE
Provide playlists for countries + services? (same structure as repo)

### DISCUSSION THREAD

**Discussion by loucasal** [iptv-org/community#755] [Ideas]
I originally mentioned this in a comment to iptv-org/iptv-org.github.io#97, but I don't think it got enough visibility there.

Currently, the repo uses a country + service (e.g. US / Pluto) structure for sorting channels, but this is not made available to users as part of the auto-generated playlists. The idea would be to provide this for users who want a more granular breakdown than offered by (very long) country lists. It doesn't need to be the default, but it can be if the community supports it.

The rationale for this is twofold: (1) grouping related channels together and (2) ensuring larger country lists have a more manageable size, as also mentioned in iptv-org/iptv-org.github.io#29. Also, if it is useful to keep the repository tidy, why wouldn't it be for users?

**Comment by what-a** [comment]
I believe you could use m3u4u to generate the desired result without any need to change IPTV-org.

**Reply by loucasal** [reply]
Sure (in fact I'm already doing it), but it is a lot of manual tinkering and changes upstream often lead to losing modifications.

**Reply by what-a** [reply]
> Sure (in fact I'm already doing it), but it is a lot of manual tinkering and changes upstream often lead to losing modifications.

Understood. Sounds like you're deeper into using it than I am. I was hoping the end result would be easier to achieve.

**Comment by freearhey** [comment]
You can already use all the playlists from the repository directly. Just click on the “Raw” button in the upper right corner:

![image](https://github.com/user-attachments/assets/3a1dca7a-20cb-4803-9012-6dbca2c420cb)
